docs/Usage/MobileCore.md

```diff
@@ -0,0 +1,549 @@
+# Mobile Core API Usage
+
+This document details the APIs provided by MobileCore, along with sample code snippets on how to properly use the APIs.
+
+For more in-depth information about the Mobile Core, visit the [official SDK documentation on Mobile Core](https://developer.adobe.com/client-sdks/documentation/mobile-core/).
+
+#### Importing Mobile Core
+
+##### Java
+
+```java
+import com.adobe.marketing.mobile.MobileCore;
+```
+
+##### Kotlin
+
+```kotlin
+import com.adobe.marketing.mobile.MobileCore
+```
+
+
+#### Getting MobileCore version
+
+##### Java
+
+```java
+final String coreVersion = MobileCore.extensionVersion();
+```
+
+##### Kotlin
+
+```kotlin
+val coreVersion: String = MobileCore.extensionVersion()
+```
+
+
+#### Registering your Application with the MobileCore
```

Comment:
Can you reword this to something like `Initializing MobileCore`?

Reply:
Done. Reworded the title and moved to before the `Registering extensions and starting the SDK` section. I also moved `getApplication()` along with this.